elro/auth.py

```diff
@@ -64,7 +71,7 @@ async def async_login(
         }
 
         async with aiohttp.ClientSession(json_serialize=dumps) as session:
-            async with session.post(URL_LOGIN, json=body, headers=headers) as resp:
+            async with session.post(BASE_UAA_URL + self._get_domain() + "/login", json=body, headers=headers) as resp:
```

Comment:
`self._get_domain()` will still block the thread, so we should make sure this part is awaited in a different thread.
For an example see: https://stackoverflow.com/questions/55027940/is-run-in-executor-optimized-for-running-in-a-loop-with-coroutines. If you need help, let me known.